<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationForm</class>
 <widget class="QWidget" name="CorrelationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="1">
       <widget class="AssignButton" name="assignButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="AvailableFieldsListView" name="availableVariables">
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="BoundListView" name="variables">
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Correlation Coefficients</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="1">
          <widget class="BoundCheckBox" name="kendallsTauB">
           <property name="text">
            <string>Kendall's tau-b</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="BoundCheckBox" name="spearman">
           <property name="text">
            <string>Spearman</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="BoundCheckBox" name="pearson">
           <property name="text">
            <string>Pearson</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Test of Significance</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="BoundGroupBox" name="tails" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="twoTailed">
              <property name="text">
               <string>Two-tailed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="oneTailed">
              <property name="text">
               <string>One-tailed</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="BoundCheckBox" name="flagSignificant">
           <property name="text">
            <string>Flag significant correlations</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonOptions">
        <property name="text">
         <string>Options</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="panelOptions" native="true">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Statistics</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="BoundCheckBox" name="meansAndStdDev">
           <property name="text">
            <string>Means and standard deviations</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BoundCheckBox" name="crossProducts">
           <property name="text">
            <string>Cross-product deviations and covariances</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QWidget" name="widget_6" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Missing Values</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="BoundGroupBox" name="missingValues" native="true">
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="excludePairwise">
              <property name="text">
               <string>Exclude cases pairwise</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="excludeListwise">
              <property name="text">
               <string>Exclude cases listwise</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundGroupBox</class>
   <extends>QWidget</extends>
   <header>widgets/boundgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOptions</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelOptions</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
